apps/authorization/management/commands/update_access_grants.py

```diff
@@ -1,27 +1,35 @@
 from django.core.management.base import BaseCommand
 from apps.authorization.models import DataAccessGrant
 from apps.dot_ext.models import Application
+
+
```

Comment:
Purposeful to match PEP 8 standard on spacing around class declarations. We should get the team using PEP 8 tools in their IDE. I fix a lot of stuff like this reflexively when I see it because PyCharm yells at me if its not to the standard.